culture/review_culture.md

```diff
@@ -0,0 +1,142 @@
+> If you want to go fast, go alone. If you want to go far, go together.
+
+As a community, we value going far over going fast. To go far, we need to stay
+aligned in our values, goals, and practices. One of the ways that we do this is
+through cultivating review culture.
+
+# On Review Culture
+
+Review culture came to us from the practices of software engineers. Large
+codebases are incredibly complex: they contain the ideas and work of hundreds
+of people, manifested in millions of lines of code. They need to remain
+logically consistent, even as they grow and change. A big part of how engineers
+manage is through code review: every single change to the system is reviewed by
+at least one other person before becoming official, or "merging".
+
+Ensuring that all work is reviewed brings several benefits. Most obviously, it
+leads to higher quality work, because the final output includes feedback and
+suggestions from others. Just as importantly, reviews enable the exchange of
+knowledge, context, and perspective within the team. In a good review, both the
+contributor and the reviewer can teach each other about how work gets done on
+the project. Finally, review ensures that every part of the project is
+understood by at least two people.
+
+# The Review Process
+
+## Changes
+The review process orients around _changes_, which are the units of work being
+reviewed.
+
+A change can take many forms, for example:
+- Adding a new guide or piece of documentation
+- Adding a new module to the codebase
+- Modifying an existing guide or module
+
+
+## Participants in the Review Process
+
+### Contributor
+
+The contributor is the person proposing a change. Anyone can decide to be a
+contributor: the only requirements are that you have a change you need to
+propose, and that you agree to be a respectful participant in the review
+process. (When we have a code of conduct--link to it here.)
+
+### Reviewer
+
+The reviewers are the people who review the change, suggest changes to it, and
+eventually approve it. Anyone in the community can act as a reviewer: the only
+requirements are that you have some insight to share (or questions to ask)
+about the change, and that you agree to be a respectful participant in the
+process.
+
+### Maintainer
+
+Maintainers are responsible for the health and maintenance of a portion of the
+project. They are the stewards of the review process, and have the privileges
+needed to move forward or block a change. Not everyone can be a maintainer; it
+requires deep knowledge of the project, high trust from the community,
+experience as both a contributor and reviewer, and substantial commitment.
+Since we invest maintainers with special privileges and trust, we have high
+expectations of them. Those acting as maintainers must be respectful,
+considerate, and judicious in the way they exercise their powers.
+
```

Comment:
Maybe something here about (general) kinds of review? Or if that is out of scope to be too much detail, we can link to something. I'm putting myself in the shoes of someone that might be new, and what I'm hungry for at this point is to be able to take this abstraction and see a concrete, written out example. E.g., "There is a repository with documentation on running software. Sigmund realizes there is an undocumented command and opens an issue on the software repository to ask if it would be useful for him to write documentation for it. He gets the green flag and checks out a branch, writes some instructions in markdown, pushes to his branch, and opens a pull request to request review for his changes. In the words of the three bears, the changes aren't too small, but aren't too big, so two reviewers is just right, so when Henny and Hunny receive a GitHub notification for the pull request, they feel like they can help out, and give Sigmund feedback on his markdown file. He is pointed to the contributing.md file, where it mentions some small details like putting one sentence per line in the file, and adding languages to code blocks for syntax highlighting. When Sigmund fixes all the suggested changes, H&H approve, and a maintainer with write access is pinged to click on the "Merge" button. After merge, the documentation is rendered live in the software web interface, and the world is right again!

Reply:
I think as a followon we should have some sort of "scope of review in SourceCred" document where we talk about what the domains of review are. For example, review of code and official docs happens on GitHub, review of weight changes (currently) happens on Discourse, review of plans to change how the Discourse is organized take place on Discourse, etc. Happy to see a pull request adding this. :)

Reply:
Tracking suggestion here https://github.com/sourcecred/docs/issues/6